internal/codeintel/codenav/service.go

```diff
@@ -929,7 +930,39 @@ func (s *Service) SCIPDocument(ctx context.Context, uploadID int, path string) (
 	return s.lsifstore.SCIPDocument(ctx, uploadID, path)
 }
 
-func (s *Service) getSyntacticUpload(ctx context.Context, repo types.Repo, commit api.CommitID, path string) (uploadsshared.CompletedUpload, error) {
+type SyntacticUsagesErrorCode = int
```

Comment:
```suggestion
type SyntacticUsagesErrorCode int
```

We don't want a type alias, we want a newtype wrapper. Yes, there are some places where Go will do implicit conversions anyways, but this will be more confusing to debug.